### DISCUSSION TITLE
How can I paste the content of docx to a Issue

### DISCUSSION THREAD

**Discussion by sky5454** [community/community#23270] [New to GitHub]
<p>when I paste the content of docx to a Issue, there only text but not any pictures. It is not so convinent.</p>
<p>Any other solutions? Thanks!</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/sky5454">@sky5454</a>!</p>
<p>Thank you for reaching out on this! <br>
Have you also tried adding a local copy of the image as an attachment into your issue?  There is an example here: “<a href="https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests" rel="nofollow noopener">File attachments on issues and pull requests</a>” if you have not already seen this method.</p>

**Comment by sky5454** [comment]
<p>I have tried, but it can only one pic by one pic attached, but not all. A docx includes lots of pics, so inconvenient. Thanks for your reply.</p>
<blockquote>
<p><a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a> wrote:</p>
</blockquote>
<blockquote>
<p>Hi <a class="mention" href="/sky5454">@sky5454</a>!</p>
</blockquote>
<blockquote>
<p>Thank you for reaching out on this! <br>
Have you also tried adding a local copy of the image as an attachment into your issue?  There is an example here: “<a href="https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests" rel="nofollow noopener">File attachments on issues and pull requests</a>” if you have not already seen this method.</p>
</blockquote>
